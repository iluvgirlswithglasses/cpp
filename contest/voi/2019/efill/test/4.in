37
-4058228 -3150160
-4057896 -3305398
-4056014 -3857030
-4055432 -3923588
-4053656 -3950454
-4046794 -4050772
-3977546 -4054702
-3885828 -4057604
-3478800 -4058084
-2096314 -4058182
3014080 -4058224
3773652 -4057422
3866412 -4055580
4030462 -4048134
4044916 -4042528
4051740 -3993898
4057898 -3856208
4058142 -3453058
4058222 -1963148
4058220 1914428
4057952 3578160
4057886 3685334
4056944 3797968
4055194 3857550
4044536 4054604
3975792 4057904
3536202 4058022
1177446 4058192
-821178 4058238
-2492490 4058240
-3680462 4057644
-4029978 4052986
-4043402 4031048
-4055480 4006512
-4057908 3779546
-4058192 3570930
-4058216 1592308
-4060884 -1908256 -4060386 -2141113
