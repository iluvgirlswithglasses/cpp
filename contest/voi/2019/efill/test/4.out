28507968.59
